src/utils.tsx

```diff
@@ -420,16 +420,40 @@ interface FilterFn {
 
 export const customFilterFunctions: Record<string, FilterFn> = {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  arrIncludesNone: (row: MRT_RowData, id: string, filterValue: any) => {
+  arrExcludesSome: (row: MRT_RowData, id: string, filterValue: any) => {
     if (Array.isArray(filterValue)) {
       return !filterValue.includes(row.getValue(id));
     }
     return row.getValue(id) !== filterValue;
   },
+  // Custom filter to check if all filter values are included
```

Comment:
```suggestion
```
Should either remove the comments from these 2 filters, or add 1 to the previous filter for consistency in my opinion.